metrics/src/metrics.rs

```diff
@@ -125,20 +125,11 @@ macro_rules! datapoint_debug {
 }
 
 lazy_static! {
-    static ref HOST_INFO: String = {
-        let v = env::var("SOLANA_METRICS_DISPLAY_HOSTNAME")
-            .map(|x| {
-                x.parse()
-                    .expect("Failed to parse SOLANA_METRICS_DISPLAY_HOSTNAME")
-            })
-            .unwrap_or(0);
-
-        let name: String = hostname().unwrap_or_else(|_| "".to_string());
-        if v == 0 {
-            hash(name.as_bytes()).to_string()
-        } else {
-            name
-        }
+    static ref HOST_ID: String = {
+        env::var("SOLANA_METRICS_HOST_ID").unwrap_or_else(|_| {
+            let hostname: String = hostname().unwrap_or_else(|_| "".to_string());
+            hash(hostname.as_bytes()).to_string()
```

Comment:
Will we never show hostname now?

Reply:
Nope, pubkey or host hash only.  I added a "host-" prefix when it's a host hash though to avoid confusion with a pubkey